src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx

```diff
@@ -328,6 +329,17 @@ function PopoverReportActionContextMenu({ref}: PopoverReportActionContextMenuPro
         policy,
     });
 
+    const [originalReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${getOriginalReportID(reportIDRef.current, reportActionRef.current)}`, {
+        canBeMissing: true,
+    });
+    const ancestorsRef = useRef<typeof ancestors>([]);
```

Comment:
why do we need this ref when we already have `ancestors` values directly available in the next line?